src/main/java/jalau/at18/katas/game2048/team3/Main.java

```diff
@@ -0,0 +1,290 @@
+package jalau.at18.katas.game2048.team3;
+
+import java.util.Scanner;
+import java.util.Random;
+
+public class Main {
+    private static final int MAX_SIZE = 4;
+    private static final int MAX_INDEX = 3;
+    private static final int ZERO = 0;
+    private static final int MIN_VAL = 0;
+    private static final int MAX_VAL = 1;
+    private static final int PRINCIPAL_NUMBER = 2;
+    private static final int TOTAL_TILES = 16;
+    private int[][] oldmatrix = new int[MAX_SIZE][MAX_SIZE];
+    private int[][] matrixG = new int[MAX_SIZE][MAX_SIZE];
+    private static final int NUMBER_WINNER = 2048;
+    public static void main(String[] args) {
```

Comment:
I found a bug, see example below, there is an unexpected "8" when I moved to left
![image](https://user-images.githubusercontent.com/4606607/184523549-510dfd7d-afd2-47ec-b5f5-2ac70756843b.png)

NOTE: you can try to write a unit test to reproduce this issue, let me know in case you cannot reproduce the same behavior

Reply:
I can't reproduce the same behavior

Reply:
weird, probably a glitch